etc/tomcat9/server-jmx.xml.patch

```diff
@@ -1,15 +0,0 @@
---- server.xml
-+++ server.xml
-@@ -46,9 +46,9 @@
-   http://tomcat.apache.org/tomcat-7.0-doc/config/listeners.html#JMX_Remote_Lifecycle_Listener_-_org.apache.catalina.mbeans.JmxRemoteLifecycleListener
-   NOTE : eXo platform Standalone Tomcat distribution bundles the required catalina-jmx-remote.jar.
-   -->
--  <!--
--  <Listener className="org.apache.catalina.mbeans.JmxRemoteLifecycleListener" rmiRegistryPortPlatform="10001" rmiServerPortPlatform="10002" useLocalPorts="false" />
--  -->
-+  
-+  <Listener className="org.apache.catalina.mbeans.JmxRemoteLifecycleListener" rmiRegistryPortPlatform="@JMX_RMI_REGISTRY_PORT@" rmiServerPortPlatform="@JMX_RMI_SERVER_PORT@" useLocalPorts="false" />
```

Comment:
It would be better to keep this in fact, the jar is embedded in Tomcat Bundle with the classes, but the Valve has to be configured.

Reply:
I tried this before, I got this error 
```bash
java.lang.ClassNotFoundException: org.apache.catalina.mbeans.JmxRemoteLifecycleListener
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at org.apache.tomcat.util.digester.ObjectCreateRule.begin(ObjectCreateRule.java:102)
	at org.apache.catalina.startup.ListenerCreateRule.begin(ListenerCreateRule.java:68)
	at org.apache.tomcat.util.digester.Digester.startElement(Digester.java:1277)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.startElement(AbstractSAXParser.java:518)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractXMLDocumentParser.emptyElement(AbstractXMLDocumentParser.java:183)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanStartElement(XMLDocumentFragmentScannerImpl.java:1386)
	at 
```
Also `catalina-jmx-remote.jar` is no longer provided since tomcat 9.0. However, when skipping applying the patch, I got JMX working properly

Reply:
Ah, ok, [deleted some days ago](https://github.com/apache/tomcat/tree/9.0.58/java/org/apache/catalina/mbeans). It seems configured by default using JRE (was intended initially to be deleted in Tomcat 10).